ui/components/app/assets/hooks/useNativeTokenBalance.tsx

```diff
@@ -81,21 +80,26 @@ export const useNativeTokenBalance = () => {
   // useCurrencyDisplay passes along the symbol and formatting into the value here
   // for sorting we need the raw value, without the currency and it should be decimal
   // this is the easiest way to do this without extensive refactoring of useCurrencyDisplay
-  const tokenFiatAmount = currencyFormatter
-    .unformat(unformattedTokenFiatAmount, {
+  const tokenFiatAmount = currencyFormatter.unformat(
+    unformattedTokenFiatAmount,
+    {
       code: currentCurrency.toUpperCase(),
-    })
-    .toString();
+    },
+  );
 
-  const nativeTokenWithBalance: TokenWithBalance = {
-    address: '',
+  const nativeTokenWithBalance = {
+    chainId: chainId as Hex,
+    address: '' as Hex,
```

Comment:
Should we update the types for `chainId` and `address` in this PR or the next? I was thinking it would make sense to handle this during integration.